geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -310,4 +323,40 @@ public void setValue1(Value1 value1) {
       this.value1 = value1;
     }
   }
+
+  @Test
+  public void testSimpleQueryWithProxyRegion() {
+    server1.invoke(() -> setupReplicatedProxyRegion(DATA_REGION_WITH_PROXY_NAME));
+    server2.invoke(() -> setupReplicatedRegion(DATA_REGION_WITH_PROXY_NAME));
+    locator.waitUntilRegionIsReadyOnExactlyThisManyServers(DATA_REGION_WITH_PROXY_NAME_PATH, 2);
+
+    server1.invoke(() -> prepareDataForRegion(DATA_REGION_WITH_PROXY_NAME_PATH));
+
+    String member = getHostingMember();
```

Comment:
for this test, you know server1 has the data and server2 is the proxy, so you don't really need to use "describe region" to get this information, you can simply add "--member=server-1" in your query command